apps/web/playwright/fixtures/regularBookings.ts

```diff
@@ -396,7 +389,113 @@ export function createBookingPageFixture(page: Page) {
+          .filter({ hasText: `Collective${collectiveEventDescription}` })
+          .getByRole("radio")
+          .first()
+          .click();
+      }
+
+      if (options.isRoundRobinType) {
+        await page
+          .locator("div")
+          .filter({ hasText: `Round Robin${roundRobinEventDescription}` })
+          .getByRole("radio")
+          .nth(1)
+          .click();
+      }
+
+      if (options.isManagedType) {
+        await page
+          .locator("div")
+          .filter({ hasText: `Managed Event${managedEventDescription}` })
+          .getByRole("radio")
+          .last()
+          .click();
+
+        const managedEventClarification = (await localize("en"))("managed_event_url_clarification");
+        await expect(page.getByText(managedEventClarification)).toBeVisible();
+      }
+
+      const continueText = (await localize("en"))("continue");
+      const toast = page.getByTestId("toast-success");
 
+      await page.getByRole("button", { name: continueText }).click();
+      expect(toast).toBeTruthy();
+      await page.getByTestId("update-eventtype").click();
+    },
+    removeManagedEventType: async () => {
+      await page
+        .locator("header")
+        .filter({ hasText: "Test Managed Event TypeSave" })
+        .getByRole("button")
+        .first()
+        .click();
+
+      // Check if the correct messages is showed in the dialog
+      const deleteManagedEventTypeDescription = (await localize("en"))(
+        "delete_managed_event_type_description"
+      );
+      const confirmDeleteEventTypeText = (await localize("en"))("confirm_delete_event_type");
+      await expect(page.getByText(deleteManagedEventTypeDescription)).toBeVisible();
+      await page.getByRole("button", { name: confirmDeleteEventTypeText }).click();
+
+      // Check if the correct image is showed when there is no event type
+      await expect(page.getByTestId("empty-screen")).toBeVisible();
+    },
+    assertManagedEventTypeDeleted: async () => {
+      const eventTypeDeletedText = (await localize("en"))("event_type_deleted_successfully");
+      await expect(page.getByRole("button", { name: eventTypeDeletedText })).toBeVisible();
+    },
+    deleteTeam: async () => {
+      const teamsText = (await localize("en"))("teams");
+      const teamLogoText = (await localize("en"))("team_logo");
+      const disbandTeamText = (await localize("en"))("disband_team");
+      const confirmDisbandTeamText = (await localize("en"))("confirm_disband_team");
+      await page.getByRole("link", { name: teamsText }).click();
+      await page.getByRole("link", { name: `${teamLogoText} Test Team` }).click();
+      await page.getByRole("button", { name: disbandTeamText }).click();
+      await page.getByRole("button", { name: confirmDisbandTeamText }).click();
+
+      // Check if the correct image is showed when there is no team
+      await expect(page.getByRole("img", { name: "Cal.com is better with teams" })).toBeVisible();
```

Comment:
It's better to add data-test-id because making any small change would keep breaking tests